server/datastore/mysql/hosts.go

```diff
@@ -755,6 +755,11 @@ const hostMDMSelect = `,
 			WHEN hmdm.enrolled = 0 AND hmdm.installed_from_dep = 0 THEN 'Off'
 			ELSE NULL
 		END,
+		'dep_profile_error',
+		CASE 
+			WHEN hdep.assign_profile_response = '` + string(fleet.DEPAssignProfileResponseFailed) + `' THEN CAST(TRUE AS JSON)
+			ELSE CAST(FALSE AS JSON)
```

Comment:
not super important: how tricky would it be to make this a placeholder?

Reply:
Seems like it would require reworking how the subclauses are being stitched into queries in a few places. Right now, that's being done with basic string concatenation. To make it a placeholder, seems like we'd need to turn this `const` into a function that returns the statement and the placeholder arg separately and then track that keep track of that in all the where it is used to make sure this arg ends up at the right index in the resulting args for the full statement. It's not super tricky, but definitely a chore now and then one more thing to keep track of when using this subclause later.